API/models/model.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../../db/connection");
+const format = require("pg-format");
```

Comment:
You've required this a devDependancy but are using it as a standard one...if you wish to stick to the implementation below this will break your build at hosting. Either refactor the function below, or install this as a standard dependancy. I'd prefer you refactor but it is your repo.